src/content/blog/ddev-name-resolution-wildcards.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "DDEV Name Resolution, Wildcards, Working Offline"
+pubDate: 2024-03-27
+#modifiedDate: 2024-03-03
+summary: How hostname name resolution works in a DDEV environment, including how wildcards work
+author: Randy Fay
+featureImage:
+  src: /img/blog/2024/03/2024-ddev.png
+  alt: Hostname resolution with DDEV
+  credit: 'Ideogram.ai: the words "2024" and "DDEV" next to each other'
+categories:
+  - Guides
+---
+
+A very cool thing about DDEV is that you don’t need to think about IP addresses or HTTPS or name resolution most of the time. Some of you will never need to. And you don't have to understand teh rest of this article to use DDEV.
+
+This article attempts to unwind what DDEV does for you, and what to do when things require a bit more effort.
+
+## Name resolution for browsers and URLs
+
+The first thing to understand is the structure of a URL used by your browser. A URL like `https://www.google.com/`/search?q=nothing` is structured with 3 main parts, the protocol (usually `https`), the hostname (like `www.google.com`), and the URI, (like `/search?q=nothing`). Every URL that your browser can visit has at least the protocol and the hostname. Sometimes the protocol is hidden implying the default `https`. The URI may be the default "front page".
```

Comment:
```suggestion
The first thing to understand is the structure of a URL used by your browser. A URL like `https://www.google.com/`/search?q=nothing` is divided into three main parts, the protocol (usually `https`), the hostname (like `www.google.com`), and the URI, (like `/search?q=nothing`). Every URL that your browser can visit has at least the protocol and the hostname. Sometimes the protocol is hidden implying the default `https`. The URI may be the default "front page".
```
i thought maybe divided is the better fit here? structured "with" sound odd anyway and i would always write out numbers between one and ten.